app/controllers/quotes_controller.rb

```diff
@@ -45,6 +45,11 @@ def destroy
     redirect_to quotes_path
   end
 
+  def best
+    @quotes = Quote.all.sort_by(&:score).reverse
```

Comment:
Although it produces the same SQL, you can omit the `all`...because typing less is best.

```
Quote.order(score: :desc)
```